DistributedWebApp/DistributedWebAppAPI/DistributedWebAppAPI.csproj

```diff
@@ -33,5 +34,9 @@
     <TypeScriptSourceRoot />
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
Niepotrzebny zapis, 1591 jest przydatny.

Reply:
racja,dodałem to przykładowo, testowo, do wywalenia;)